### ISSUE TITLE
[Feature Request] Apply Effect

### ISSUE THREAD

**Issue by jelaw21** [tmetcalfe89/interactions#1]
Any chance we can get a boolean for whether to not to apply the bonemeal animation to give our magical interactions a bit more flair?

**Comment by tmetcalfe89-zz** [comment]
How about a variable for the particle type and another for the amount of particles to spawn?
I believe that's how vanilla bonemeal works, I'll check it out!

**Comment by jelaw21** [comment]
sweet, that would be cool!

**Comment by tmetcalfe89-zz** [comment]
Complete. Just waiting for CurseForge to upload and we'll be good to go!

**Comment by jelaw21** [comment]
C'mon CurseForge hurry up!  Also, is there a list somewhere of particle
effects?

**Comment by tmetcalfe89-zz** [comment]
The delay is my fault. I uploaded 1.2 as an alternative to 1.1 by accident. I've submitted a ticket to have it switched to its own release version, but it was over the weekend so I didn't expect any movement on it until today. I'll keep my eyes on their response and get that fixed as soon as possible.

I've added a list of available particle effects [here](https://github.com/tmetcalfe89/interactions/blob/master/ParticleTypes.txt).
